BuildingSync.xsd

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:auc="http://buildingsync.net/schemas/bedes-auc/2019" targetNamespace="http://buildingsync.net/schemas/bedes-auc/2019" elementFormDefault="qualified" attributeFormDefault="unqualified">
   <xs:annotation>
-    <xs:documentation>BuildingSync Schema - Version 1.1-prerelease</xs:documentation>
+    <xs:documentation>BuildingSync Schema - Version 2.0-prerelease</xs:documentation>
```

Comment:
Consider moving this change to a separate pull request.

Reply:
valid concern. moving.